### PR TITLE
move_operate-first.github.io-to-old

### PR DESCRIPTION
**Pull request by durandom**
in preparation to launch the new website
https://github.com/operate-first/website-ng/issues/29

### CODE REVIEW

**Comment by goern** [comment]
Will there be a new `operate-first.github.io` ?

**Comment by durandom** [comment]
yes. the website will relaunch. it's all in the linked issue :)

File: github-config.yaml [hunk 1]
```diff
@@ -209,6 +209,13 @@ orgs:
       operate-first.github.io:
         has_projects: false
         has_wiki: false
+        previously:
+          - website-ng
+      operate-first.github.io-old:
+        has_projects: false
+        has_wiki: false
+        previously:
```
[the remark this answers is not in the document]
Comment (durandom) [reply github-config.yaml]: done

**Comment by HumairAK** [comment]
Not sure why it's failing: 
```
{"component":"peribolos","file":"k8s.io/test-infra/prow/cmd/peribolos/main.go:209","func":"main.main","level":"fatal","msg":"Configuration failed: failed to configure operate-first teams: failed to update blog members: failed to list blog(blog) members: return code not 2XX: 404 Not Found","severity":"fatal","time":"2022-05-19T14:29:03Z"}
{"component":"entrypoint","error":"wrapped process failed: exit status 1","file":"prow/entrypoint/run.go:80","func":"k8s.io/test-infra/prow/entrypoint.Options.Run","level":"error","msg":"Error executing test process","severity":"error","time":"2022-05-19T14:29:03Z"}
```

says it can't list blog members but that's not even being updated here, overriding: 

/override peribolos
/lgtm
/approve

**Pull request merged**